packages/design-system/src/__theme__/default/index.css

```diff
@@ -111,23 +119,27 @@
   --ads-v2-color-border-muted: var(--ads-v2-color-gray-200);
 
   /** Used as active border when --ads-v2-color-bg-brand is the default bg */
-  --ads-v2-color-border-brand-emphasis-plus: var(--ads-v2-color-orange-800);
+  --ads-v2-color-border-brand-emphasis-plus: var(--ads-v2-color-orange-500);
   /** Used as hover border when --ads-v2-color-bg-brand is the default bg */
-  --ads-v2-color-border-brand-emphasis: var(--ads-v2-color-orange-700);
+  --ads-v2-color-border-brand-emphasis: var(--ads-v2-color-orange-400);
   /** Used as active border hover color of secondary items like radio checkbox etc */
   --ads-v2-color-border-brand-secondary-emphasis: var(--ads-v2-color-gray-700);
   /** Used as active border color of secondary items like radio checkbox etc */
   --ads-v2-color-border-brand-secondary: var(--ads-v2-color-gray-600);
   /** Used as default border color when --ads-v2-color-bg-brand is the default bg */
-  --ads-v2-color-border-brand: var(--ads-v2-color-orange-600);
+  --ads-v2-color-border-brand: var(--ads-v2-color-orange-300);
```

Comment:
This should be 500?

Reply:
It looks like this is the value in the semantic tokens list.
https://www.notion.so/appsmith/Semantic-Color-Token-174c8017d8e5481b833805c22345f493?pvs=4#b83feb4239a0479c86c6c25b82710f45

cc @vasanthappsmith is there a change that needs to be made here?

Reply:
@vasanthappsmith, we need a border for the primary button because these are the designs you gave us when we were building the components during the ADS 2.0 project phase one. 

We will need to keep in mind that semantic tags are used everywhere, including many places in the main repo we don't have control over.